src/applications/caregivers/tests/components/PreSubmitInfo/index.spec.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+
+import {
+  veteranSignatureContent,
+  primaryCaregiverContent,
+  secondaryCaregiverContent,
+} from 'applications/caregivers/definitions/content.js';
+
+describe('SignatureBox content', () => {
+  it('veteranSignatureContent has not changed', () => {
+    const mockVeteranSignatureContent = [
+      'I certify that I give consent to the individual(s) named in this application to perform personal care services for me upon being approved as Primary and/or Secondary Family Caregivers in the Program of Comprehensive Assistance for Family Caregivers.',
+    ];
+
+    expect(mockVeteranSignatureContent).to.deep.equal(veteranSignatureContent);
```

Comment:
UTs should test program logic, not variable or config values. lmk if i can help

Reply:
After Mike and I spoke on the phone of why this implementation exist we decided to move this test to Cypress for a more scalable testing solution.